app/globals.css

```
@@ -121,6 +121,26 @@
   }
   body {
     @apply bg-background text-foreground transition-colors duration-300;
+    background-image:
+      radial-gradient(circle at top, color-mix(in oklab, var(--accent) 10%, transparent), transparent 42%),
+      linear-gradient(to bottom, color-mix(in oklab, var(--background) 96%, white 4%), var(--background));
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)